features/profile/view/Overview.tsx

```diff
@@ -1,10 +1,10 @@
 import Alert from "components/Alert";
 import Button from "components/Button";
+import UserOverview from "components/UserOverview/UserOverview";
```

Comment:
```suggestion
import UserOverview from "components/UserOverview";
```

Since you added the index file